tap_frontapp/discover.py

```diff
@@ -0,0 +1,38 @@
+import singer
+from singer import metadata
+from singer.catalog import Catalog, CatalogEntry, Schema
+from .schemas import get_schemas
+
+LOGGER = singer.get_logger()
+
+
+def discover() -> Catalog:
```

Comment:
To keep consistency with existing implementation, please remove `type annotation` from the new implementation.